src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
OK, I see. This is a delicate situation; re-using elements that had various operations applied to them before and using them for support tests may break support tests if done in a particular order; I'd rather take the slightly larger size and not risk introducing hard to track race conditions.

At least when we had one big `support.js` file it all happened in a stable order; now it depends on the methods used by a jQuery user.